lib/Models/CatalogItem.js

```diff
@@ -809,6 +875,36 @@ CatalogItem.prototype.useClock = function() {
     });
 };
 
+/**
+ * Move the current time to the next time interval.
+ */
+CatalogItem.prototype.nextTime = function() {
```

Comment:
Maybe something like `moveToNextTime`? Function names should be verbs.

Reply:
Done 8920ba9db1d99cd9c0d16f9f24b340201cc6d107